src/components/admin/arts/art-form-field.tsx

```diff
@@ -0,0 +1,145 @@
+import FormField from '@/components/admin/form-field';
+import {
+  Select,
+  SelectTrigger,
+  SelectValue,
+  SelectContent,
+  SelectItem,
+} from '@/components/ui/select';
+import { Search, X } from 'lucide-react';
+import { Gallery } from '@/types';
+import { useCallback } from 'react';
+
+interface Props {
+  form: any;
+  galleries: Gallery[];
+  onChange: (
+    e: React.ChangeEvent<HTMLInputElement | HTMLTextAreaElement>
+  ) => void;
+  onSelectGallery: (galleriesNo: number) => void;
+  onChangeArtType: (value: 'SINGLE' | 'GROUP') => void;
+  onClickArtistModal: () => void;
+  onRemoveArtist: (artistNo: number) => void;
+  onChangeArtistDescription: (artistNo: number, value: string) => void;
+}
+
+export default function ArtworkFormFields({
+  form,
+  galleries,
+  onChange,
+  onSelectGallery,
+  onChangeArtType,
+  onClickArtistModal,
+  onRemoveArtist,
+  onChangeArtistDescription,
+}: Props) {
+  const handleArtistDescChange = useCallback(
+    (artistNo: number, value: string) => {
+      onChangeArtistDescription(artistNo, value);
+    },
+    [onChangeArtistDescription]
+  );
+
+  return (
+    <div className='w-full max-h-[335px] overflow-y-auto border border-btn-gray-d rounded overflow-hidden divide-y divide-btn-gray-d'>
+      <FormField id='artType' label='작품 유형'>
+        <Select
+          value={form.artType}
+          onValueChange={(value) =>
+            onChangeArtType(value as 'SINGLE' | 'GROUP')
+          }
+        >
+          <SelectTrigger className='w-full border-none bg-white/0 px-[15px] outline-none'>
+            <SelectValue />
+          </SelectTrigger>
+          <SelectContent>
+            <SelectItem value='SINGLE'>SINGLE</SelectItem>
+            <SelectItem value='GROUP'>GROUP</SelectItem>
+          </SelectContent>
+        </Select>
+      </FormField>
+
+      <FormField id='artName' label='작품명'>
+        <input
+          name='artName'
+          value={form.artName}
+          onChange={onChange}
+          className='w-full px-[15px] outline-none'
+        />
+      </FormField>
+
+      <FormField id='caption' label='캡션'>
+        <input
+          name='caption'
+          value={form.caption}
+          onChange={onChange}
+          className='w-full px-[15px] outline-none'
+        />
+      </FormField>
+
+      <FormField id='galleriesNo' label='전시회'>
+        <Select
+          value={String(form.galleriesNo)}
+          onValueChange={(value) => onSelectGallery(Number(value))}
+        >
+          <SelectTrigger className='w-full border-none bg-white/0 px-[15px] outline-none'>
+            <SelectValue />
+          </SelectTrigger>
+          <SelectContent>
+            {galleries.map((g) => (
+              <SelectItem key={g.galleriesNo} value={String(g.galleriesNo)}>
+                {g.title}
+              </SelectItem>
+            ))}
```

Comment:
조금 더 의미있는 변수명을 사용하면 코드 가독성이 향상됩니다!

```suggestion
            {galleries.map((gallery) => (
              <SelectItem key={gallery.galleriesNo} value={String(gallery.galleriesNo)}>
                {gallery.title}
              </SelectItem>
            ))}
```